ess/src/API/EMBC.ESS/Resources/Suppliers/Contract.cs

```diff
@@ -46,6 +46,7 @@ public class Supplier
         public SupplierContact Contact { get; set; }
         public Team Team { get; set; }
         public bool IsPrimarySupplier { get; set; }
+        public bool MutualAid { get; set; }
```

Comment:
I'm not sure why both primary supplier flag and mutual aid flag are needed, TMK they are mutually exclusive